docs/topics/simulation_setup.rst

```diff
@@ -0,0 +1,182 @@
+helper ensembles, such as the minus ensemble). In multiple state systems,
+TIS can involve hundreds of ensembles.
+
+Monte Carlo methods (configurational or path sampling) draw samples from a
+statistical ensemble. For configurational sampling, the thermodynamic
+ensemble to be sampled is defined by the underlying Hamiltonian of the
+system (which includes the masses of the particles and the interaction
+energies) and the natural variables of the ensemble (e.g., for the
+canonical, or :math:`NVT`, ensemble, these are the number of particles, the
+volume, and the temperature).  Similarly, a path ensemble will be defined by
+the underlying dynamics (including the underlying thermodynamic ensemble)
+and additional restrictions (such as state definitions).
+
+Selecting the extra restrictions for path sampling is much more difficult
+that selecting the thermodynamic state to sample: whereas the temperature
+and density usually come from experimental considerations, the volume of
+phase space that defines a stable state is not an experimentally observable
+quantity. Typically, defining the state requires performing MD simulations
+in each stable state to characterize its bounds, and there is not a unique
+appropriate definition when studying a particular problem of interest.
+
+In TPS, only the states need to be defined. In TIS, one must also define the
+interfaces.
+
+Defining good states
+--------------------
+
+In TPS, the main traits defining the path ensemble are the state
+definitions. The essential point is that states should be stable: a
+trajectory started from any point in the state should be expected to take a
+long time before visiting any other state.
+
+The topic of state definitions has been included in several review articles.
+For more, see:
+
+* `Bolhuis and Dellago. "Trajectory-based rare event simulations." in
+  Reviews Computation Chemistry vol. 27 (2010)
+  <http://onlinelibrary.wiley.com/doi/10.1002/9780470890905.ch3/summary>`_
+
+In TIS, the role of the minus move in decorrelating trajectories should also
+be considered. Ideally, a single minus move trajectory should be just long
+enough to fully decorrelate (lose all memory) within the state. This puts
+restrictions on both the volume of the state as well as that of the
+innermost interface.
+
+
+Defining good interfaces
+------------------------
+
+Much of the literature on TIS uses the expression that "interfaces foliate
+phase space." In the OPS picture, where interfaces are volumes instead of
+surfaces, this means that inner interfaces (closer to the state) must be
+fully contained by outer interfaces. In other words, for every volume in the
+ordered set of the state, interface 0, interface 1, etc.; any point in phase
+space that is inside interface :math:`i` is also inside interface :math:`i+1`.
+Importantly, this includes the state: the innermost interface must entirely
+encapsulate the state. Note that there is no way for OPS to check this for
```

Comment:
Technically there is a way. If a configuration inside a volume is also inside all higher interface volumes, for all configurations, than the definition is correct. This could be coded, and a fault given when this is not obeyed.

Reply:
Adjusted to just say that we don't do it.

The better way might actually be for the state to be defined as `state | innermost_interface`, which does formally include this, but makes the `network` objects more confusing and means that the state that is sampled may not be what the user requested.

If using our standard InterfaceSet objects, I think that successive interfaces (not including the state) should be forcibly correct in this regard.